src/test/java/core/basesyntax/FileReaderTest.java

```diff
@@ -0,0 +1,87 @@
+package core.basesyntax;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotEquals;
+import static org.junit.Assert.assertTrue;
+
+import com.opencsv.exceptions.CsvException;
+import core.basesyntax.service.FileReaderService;
+import core.basesyntax.service.impl.FileReaderImpl;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FileReaderTest {
+    private static final String VALID_FILE = "src/test/resources/input.csv";
+    private static final String INVALID_FILE_HEAD = "src/test/resources/input2.csv";
+    private static final String WRONG_FILE = "src/test/resources/wrong.csv";
+    private static final String ABSENT_FILE = "src/test/resources/input23.csv";
+    private static FileReaderService fileReaderService;
+
+    @BeforeClass
+    public static void beforeAll() {
+        fileReaderService = new FileReaderImpl();
+    }
+
+    @Test
+    public void readRightFile_OK() throws IOException, CsvException {
```

Comment:
name of the method should be like this pattern `<methodUnderTest>_<state>_<expectedBehavior>`. For example for this case it'll be `read_validFile_ok()`